conf/longranger.config

```diff
@@ -0,0 +1,22 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running longranger
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+if (("$PROFILE" == "singularity") && (System.getenv('GITHUB_ACTION') == null)) {
+    singularity {
+        // Enter here the necessary configuration for the Singularity container
+        // to see your HPC cluster. These options will have to be inserted in the
+        // pipeline's configuration.
+        // Below is a working example for LSF at the Sanger institute. The file
+        // named martian.lsf.template has to be shipped with the pipeline (update
+        // its relative path to the pipeline's root directory ${projectDir}), and
+        // possibly change the path of the singularity binary.
+        envWhitelist = "LSF_BINDIR,LSF_SERVERDIR,LSF_LIBDIR,LSF_ENVDIR"
+        runOptions = "-B ${projectDir}/martian.lsf.template:/opt/longranger-2.2.2/martian-cs/2.3.2/jobmanagers/lsf.template -B /software -B /etc/lsf.conf --env APPEND_PATH=$LSF_BINDIR:$LSF_SERVERDIR:/software/singularity-v3.9.0/bin"
+    }
+}
+
+singularity.autoMounts = true
```

Comment:
This will automatically come from `-profile singularity` so can be removed here.